src/TLuaInterpreter.cpp

```diff
@@ -12531,6 +12531,67 @@ int TLuaInterpreter::getCmdLine(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#addCmdLineSuggestion
+int TLuaInterpreter::addCmdLineSuggestion(lua_State* L)
```

Comment:
Because these are not saved, they act just like the temp items. How about calling them `addTempSuggestion()` and the rest?

Reply:
I was thinking like that... but command line or windows are not saved as well and are not temp. Since this is tied to command line I left it like that. But up to you, I agree with your reasoning as well.

Reply:
That's a fair point, it's not all that clear. @demonnic @Kebap ?

Reply:
As something that might shift decision a bit - all `temp` apis have their `perm` counterparts and they all refer to basic concepts (alias, trigger, etc.) that can be added through GUI.

Reply:
Good point.